git/hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+files=`git diff --cached --name-status`
+re="gen-PKGBUILD.py"
+if [[ $files =~ $re ]]
+then
+  echo "Re-generating PKGBUILD..."
+  python gen-PKGBUILD.py > PKGBUILD
+  git add gen-PKGBUILD.py
+  echo "Re-generating .SRCINFO..."
+  makepkg --printsrcinfo > .SRCINFO
+  git add .SRCINFO
+  git commit --amend -C HEAD --no-verify
```

Comment:
I think you can remove the amend and since it's a pre-commit hook these things will be commited properly.  It seems to work for me anyway.